Synergy/Validators/SharedValidatorLogic/CustomGroupedValidators.cs

```diff
@@ -0,0 +1,24 @@
+using System.Linq.Expressions;
+using FluentValidation;
+
+namespace Synergy.Validators.SharedValidatorLogic;
+
+public class CustomGroupedValidators<T> : AbstractValidator<T>
+{
+    protected void ValidateNullEmptyAndHarmfulCharacters(Expression<Func<T, string>> propertySelector,
+        string errorMessage)
+    {
+        RuleFor(propertySelector).NotEmpty().NotNull().Must(text => !text.ContainsHarmfulCharacters())
```

Comment:
I think these would be best as custom validators rather than this method due to readability and re-usableability.